index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+//Connect to Database:
+
+
+$db = new PDO('mysql:host=db; dbname=skateplazas', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `name`, `country`, `city`, `dob`, `status`, `vibe`, `photo`
+FROM `skateplazas`;");
+$query->execute();
+$skatePlazas = $query->fetchAll();
```

Comment:
I'd then suggest putting this into another function for `getAllPlazas` or something like that, that runs the DB query and returns all of the collection items